scripts/case.setupbuild.slurm

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#SBATCH --job-name=CESM-case-setupbuild
+#SBATCH --output="%x-%j.out"
+#SBATCH --time=4:00:00
+#SBATCH --nodes 1
+#SBATCH --ntasks 16
```

Comment:
if this is supposed to be used on Hortense, you need the project (`-A`) too?

Reply:
In that case the user has to submit the job with `sbatch -A <project> case.setupbuild.slurm`.

Reply:
maybe mention in the docs?